test/Benchmarks/src/Natural_Order_Sort.enso

```diff
@@ -0,0 +1,24 @@
+from Standard.Base import all
+
+import Standard.Test.Bench
+
+import Standard.Test.Faker
+import Standard.Base.Data.Ordering.Natural_Order
+
+## Bench Utilities ============================================================
+
+vector_size = 10000
+iter_size = 100
+num_iterations = 10
+
+
+# The Benchmarks ==============================================================
+
+main =
+    l = Faker.upper_case_letters
+    n = Faker.numbers
+    template = [l, l, l, n, n, n, n, n, l]
+    random_generator = Faker.make_generator 1644575867
```

Comment:
What is this number? Is there any reason for why it was chosen?

Reply:
I think we should give names to all magic numbers (and if necessary, a short comment).